pkg/scanner/utils/utils_test.go

```diff
@@ -0,0 +1,80 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/knqyf263/go-version"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMatchVersions2(t *testing.T) {
```

Comment:
Is there any reason that you need the suffix 2?

```suggestion
func TestMatchVersions(t *testing.T) {
```

Reply:
It was committed mistakenly, was comparing the older/new version for correctness, fixed